Sources/PackageCollections/PackageCollections.swift

```diff
@@ -294,6 +350,36 @@ public struct PackageCollections: PackageCollectionsProtocol {
             return PackageCollectionsModel.TargetListItem(target: pair.target, packages: targetPackages)
         }
     }
+
+    // TODO: confirm this is the correct logic 👀
+    internal func mergedPackageMetadata(package: PackageCollectionsModel.Collection.Package,
+                                        basicMetadata: PackageCollectionsModel.PackageBasicMetadata?) -> PackageCollectionsModel.Package {
+        var versions = package.versions.map { packageVersion in
+            PackageCollectionsModel.Package.Version(version: packageVersion.version,
+                                                    packageName: packageVersion.packageName,
+                                                    targets: packageVersion.targets,
+                                                    products: packageVersion.products,
+                                                    toolsVersion: packageVersion.toolsVersion,
+                                                    verifiedPlatforms: packageVersion.verifiedPlatforms,
+                                                    verifiedSwiftVersions: packageVersion.verifiedSwiftVersions,
+                                                    cves: nil, // FIXME: where do we get this from?
```

Comment:
@yim-lee 👀

Reply:
Good question. This would either be included as part of `PackageCollectionModel.Collection.PackageVersion` or `PackageMetadataProvider`. My original thought was it would come from `PackageMetadataProvider` or even `PackageVersionMetadataProvider`. The difference is that package collection data is somewhat static, while metadata providers can provide up-to-date data, as CVEs should be. What do people think?

Reply:
`PackageVersionMetadataProvider` sounds right to me, but I think it means we need to expand it handling of version since CVEs are tied to versions under the current model

Reply:
Yes, note `PackageMetadataProvider` vs. `PackageVersionMetadataProvider`.

Reply:
Right now we don't have a provider for version metadata, and the only information we plan to retrieve from it is CVEs. Maybe we should remove `cves` from the model until we are ready to support it.

Reply:
okay I addressed in the data model and we can further modify when we implement the actually providers